pkg/antctl/transform/addressgroup/transform.go

```diff
@@ -30,11 +35,28 @@ type Response struct {
 }
 
 func listTransform(l interface{}, opts map[string]string) (interface{}, error) {
-	groups := l.(*cpv1beta.AddressGroupList)
-	result := []interface{}{}
-	for i := range groups.Items {
-		item := groups.Items[i]
-		o, _ := objectTransform(&item, opts)
+	groupsList := l.(*cpv1beta.AddressGroupList)
+	if len(groupsList.Items) == 0 {
+		return "", nil
+	}
+	sortField := opts["sort-by"]
+	if sortField == "" {
+		sortField = ".metadata.name"
+	}
+
+	var addressGroupRuntime runtime.Object = &cpv1beta.AddressGroupList{
+		Items: groupsList.Items,
+	}
+	addressGroupRuntimeObjectList, _ := meta.ExtractList(addressGroupRuntime)
```

Comment:
I don't think earlier comments have been addressed correctly. It feels like this should simply be:
```go
addressGroupRuntimeObjectList, _ := meta.ExtractList(groupsList)
```